include/enums.h

```diff
@@ -1275,6 +1296,17 @@ enum SoundIDs {
     SOUND_8000006A                  = 0x8000006A,
     SOUND_8000006B                  = 0x8000006B,
     SOUND_8000006C                  = 0x8000006C,
+
+    SOUND_FireBar0                  = 0xB0000000,
+    SOUND_FireBar1                  = 0xB0000001,
+    SOUND_FireBar2                  = 0xB0000002,
+    SOUND_FireBar3                  = 0xB0000003,
+    SOUND_FireBar4                  = 0xB0000004,
+    SOUND_FireBar5                  = 0xB0000005,
+    SOUND_FireBar6                  = 0xB0000006,
+    SOUND_FireBar7                  = 0xB0000007,
+    SOUND_FireBar8                  = 0xB0000008,
+    SOUND_FireBar9                  = 0xB0000009,
```

Comment:
I'd rather capitalise these like `SOUND_FIRE_BAR_N` to be consistent

Reply:
My bad, good catch.